testworld/park_test.go

```diff
@@ -46,3 +46,70 @@ func TestHost_Happy(t *testing.T) {
 	getInvoiceAndCheck(eCharlie, docIdentifier, "GBP")
 	fmt.Println("Host test success")
 }
+
+func TestHost_AddExternalCollaborator(t *testing.T) {
+	alice := doctorFord.getHost("Alice")
+	bob := doctorFord.getHost("Bob")
+	charlie := doctorFord.getHost("Charlie")
+	eAlice := alice.createHttpExpectation(t)
+	eBob := bob.createHttpExpectation(t)
+	eCharlie := charlie.createHttpExpectation(t)
+
+	a, err := alice.id()
```

Comment:
I just checked the definition

```
return h.identity.CentID(), nil
```

Why have `error` return when we are returning nil always @vimukthi-git ?

Reply:
its just to comply to the interface as it might change in the future inside the function, if will avoid nasty surprises.